shell/packages/sandstorm-db/db.js

```diff
@@ -56,6 +55,11 @@ if (Meteor.isServer && process.env.LOG_MONGO_QUERIES) {
 //       verifiedEmail: Only provided by some services. Cannot be directly edited by the user.
 //       main: True if this is the user's main identity.
 //       noLogin: True if the user does not trust this identity for account authentication.
+//       devName: Only present when `service` is "dev". A string containing the name used to create
+//                this dev account.
+//       emailTokens: Only present when `service` is "emailToken". An array of objects of the form
```

Comment:
I like that.

Though I'd say we shouldn't share e-mail addresses with contacts by default. Possibly arguably those fields shouldn't be rolled into profile? Not sure.

If we're going to do this, we should do it before the next release to avoid redundant migrations. Not sure what to do about our personal installs, though.